### PR TITLE
Documentation: Add information about installation on FreeBSD

### PR DESCRIPTION
**Pull request by amotl**
Dear @dlangille,

thanks again for your efforts at GH-646. Can I humbly ask you to proof-read the new documentation section about installing mqttwarn on FreeBSD? Any suggestions are very welcome.

With kind regards,
Andreas.

### CODE REVIEW

File: docs/usage/freebsd.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+(using-freebsd)=
+# Installing mqttwarn on FreeBSD
+
+With this installation method, you will acquire the [py-mqttwarn] package from
+the FreeBSD ports tree and install it on your machine.
+
+## Synopsis
+
+```bash
+pkg install py-mqttwarn
```
Comment (dlangille) [inline docs/usage/freebsd.md:10]: The package name would be something like `py39-mqttwarn` so the name will vary.

Using the package `origin` always worksL `pkg install sysutils/py-mqttwarn`

Comment (amotl) [reply docs/usage/freebsd.md]: Thank you. I've just update the patch according to your suggestions.

**Review by jpmens — APPROVED** [review]

File: docs/usage/freebsd.md (hunk 1 above)
Comment (amotl) [reply docs/usage/freebsd.md]: Hi. I am merging this now, hoping that my changes have been accurate.

Comment (amotl) [reply docs/usage/freebsd.md]: The documentation is now available at https://mqttwarn.readthedocs.io/en/latest/usage/freebsd.html. Enjoy, and feel free to tell us any suggestions.

If you still have the energy to support us further, I could imagine dropping a corresponding service wrapper file template into the folder at [^1]. If you can provide a pointer where I could derive a good blueprint from, I will submit a corresponding patch to mqttwarn.

[^1]: https://github.com/jpmens/mqttwarn/tree/main/etc

**Pull request merged**